tests/unit/AlterationSummary.spec.ts

```diff
@@ -143,4 +153,18 @@ describe('Alteration Summary component', () => {
     expect(divs.at(1).text()).toContain('The alteration for this business will be effective as of:')
     expect(divs.at(1).text()).toContain('Friday, March 5, 2021 at 8:30 am Pacific time')
   })
+
+  it('displays the list of name translations', async () => {
+
+    // Verify list exists
+    expect(wrapper.find('#name-translation').exists()).toBeTruthy()
+
+    // Verify list items
+    const namesList = wrapper.vm.$el.querySelectorAll('.info-text')
+    expect(namesList[0].textContent).toContain(nameTranslationsList[0].name)
+    expect(namesList[1].textContent).toContain(nameTranslationsList[1].name)
+    expect(namesList[2].textContent).toContain(nameTranslationsList[2].name)
+    expect(namesList[3].textContent).toContain(nameTranslationsList[3].name)
```

Comment:
These should all be capitalized right?

Reply:
You would think, but the text appears on the page capitalized via css vs the actual value. But since we have "toUppercase" now, the value will also be capitalized